### DISCUSSION TITLE
BaseValidator.Test not found

### DISCUSSION THREAD

**Discussion by csunkara-ipipeline** [spring-projects/spring-net#236] [Q&A]
I am currently looking at upgrading a decade old project that uses the old Spring.Core dll and since we are now upgrading to .Net 7 I found the nuget package as a logical replacement. However, we have a code that's using 'Test' property under BaseValidator which I cannot find anymore in the nuget package. Is there a suitable replacement for this?

**Comment by csunkara-ipipeline** [comment]
Can anyone help with this?

**Comment by lahma** [comment]
If you search the `BaseValidator` history, you can find the commit that made the change: https://github.com/spring-projects/spring-net/commit/6091e9c491ca77de5ccbc8efa04772b9fecb42ef#diff-4a8dccfff058d91a54b7be1a28b06c9e7dcfa054d44d912383e8f87a7f841093 . Now that you know what has changed you can plan your changes accordingly.
